doc/user/content/sql/create-source/load-generator.md

```diff
@@ -99,7 +101,7 @@ views:
     `amount`     | [`bigint`]                   | The bid amount in dollars.
     `bid_time`   | [`timestamp with time zone`] | The time at which the bid was placed.
 
-The organizations, users, and accounts are fixed at the time the auction source
+The `organizations`, `users`, and `accounts` are fixed at the time the source
```

Comment:
nit: I wrote this meaning the non-monospace organizations—i.e., the actual organizations entities, not the table itself. If you want to switch to refer to the subsources I think you either need to drop the `The` or add `subsources`:

```suggestion
`organizations`, `users`, and `accounts` are fixed at the time the source
```

```suggestion
The `organizations`, `users`, and `accounts` subsources are fixed at the time the source
```